packages/e2e/src/specs/contacts/imports-vcf-google-outlook.e2e.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+const path = require("path")
+import NavigationTabs from "../../page-objects/tabs.page"
+import ContactsPage from "../../page-objects/contacts.page"
+import ModalGeneralPage from "../../page-objects/modal-general.page"
+import modalContactsPage from "../../page-objects/modal-contacts.page"
+
+describe("import vcf, google, outlook", () => {
```

Comment:
1. How about `google` and `outlook` imports? If they are going to be added later, for now I would just change `import vcf, google, outlook` to `import vcf`.
2. I would consider changing name of this file to e.g. `import-contacts.e2e.ts`. No need to change the name in future if e.g. we decide to add import from other sources.